cmd/gitserver/server/patch.go

```diff
@@ -217,7 +217,7 @@ func (s *Server) createCommitFromPatch(ctx context.Context, req protocol.CreateC
 		committerEmail = authorEmail
 	}
 
-	cmd = exec.CommandContext(ctx, "git", "commit", "-m", message)
+	cmd = exec.CommandContext(ctx, "git", "commit", "-m", `"`+message+`"`)
```

Comment:
we can do it like this:
```suggestion
	cmd = exec.CommandContext(ctx, "git", "commit", "-m", fmt.Sprintf("%q", message))
```